assets/scripts/datalayer/common.js

```diff
@@ -425,6 +427,32 @@ document.shopAnalytics = {
     removeProductsFromCart($cart_items);
   }
 
+  /**
+   * Populates tracking data for the clicked nav item.
+   */
+  function onNavItemClick(e) {
+    var nav = null;
+    var context = '';
+
+    if (nav = this.closest('nav[aria-label]')) {
+      context = nav.getAttribute('aria-label');
+    }
+    else if (nav = this.closest('nav[id]')) {
+      context = nav.getAttribute('id');
+    }
+    else {
+      context = 'unknown';
+    }
+
+    var event_data = {
+      'event': 'UniversalEvent',
+      'eventCategory': 'User Interaction | ' + context + ' menu',
+      'eventAction': 'Click',
+      'eventLabel': this.innerText,
```

Comment:
`innerText` returns the modified, and visible text. If the label transformed by CSS it's transformed as is. We could swap this for `textContent` which will return the (full) source code text.

As discussed with @daschmidt94, `innerText` and thus e.g. the all uppercase label is preferred as this makes it easier to identify which label was clicked.

A good summary can be read in [this SO answer](https://stackoverflow.com/a/35213639/2862127)